cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,26 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import UserPageObject from '../support/pages/user.pageObject';
+
+const userPage = new UserPageObject();
+
 describe('User', () => {
   before(() => {
+    cy.task('db:clear');
+  });
+
+  it('should provide an ability to follow the another user', () => {
+    userPage.visit();
+    userPage.followUser();
 
+    cy.contains('Following').should('be.visible');
   });
 
-  it.skip('should be able to follow the another user', () => {
+  it('should provide an ability to unfollow the user', () => {
+    userPage.visit();
+    userPage.unfollowUser();
```

Comment:
Ensure that the `unfollowUser` method in `UserPageObject` is correctly implemented to unfollow the user.